src/app/modules/map/components/map/map.component.ts

```diff
@@ -376,6 +386,7 @@ export class MapComponent implements OnInit {
         });
         if (this.isOfflineMapsAvailable()) {
           layer.resampling = false; //see this.disableResamplingOfOnlineBasemap()
+          this.logger.debug(`deaktiverer resampling av fliser i lag '${layer.id}'`);
```

Comment:
Har dette noe med bugen å gjøre?

Reply:
Ja, det kan være. Deaktivering av resampling er en ganske ny funksjon og var Aslak og mitt første forsøk på å gjøre offline-fliser synlig når vi hadde dårlig dekning. Uten denne funksjonen, ble online-flisa forstørret når man zoomet inn, men så ble forstørrelsen stående og dekke over offline-flisa under, i hvert fall inntil man røk ut på en timeout. 
Logging av dette var det første jeg la til under feilsøking av RO-1301.
Er ikke sikkert vi trenger både deaktivering av resampling og kort timeout for å få offline-kart til å funke under dårlig dekning.